packages/msi/msi/Product.wxs

```diff
@@ -6,13 +6,23 @@
 
 <?define UpgradeCodeStandard = "b0fee933-ccd2-467c-8fe4-bb0ac6a099c8" ?>
 <?define UpgradeCodeEnterprise = "4ABDD5C7-E1E1-41A6-8119-DCE65634A6CC" ?>
-<?if $(var.MondooSKU) = "standard" ?>
+<?define UpgradeCodeArm64 = "090cfb7d-c00c-4d36-94fe-f649a4b29c91" ?>
+<?if $(var.arch) = "arm64"?>
+<?define UpgradeCode = "$(var.UpgradeCodeArm64)"?>
+<?define InstallerVersion="500"?>
+<?define ProductName = "Mondoo"?>
+<?define ServiceInstallStart = "demand" ?>
+<?define RegistrationTokenRequired = "0"?>
+<?define OtherSKU = "$(var.UpgradeCodeEnterprise)"?>
+<?else?>
+<?define InstallerVersion="200"?>
```

Comment:
for my own learning, what are these `InstallerVersion` for and, why these numbers `200` and `500`?¿

Reply:
apparently - I'm not expert mysel - that is the minimum MS Installer version, these are the recommendations for x64 and arm as per https://wixtoolset.org/docs/v3/xsd/wix/package/
In reality, 200 seems to be Windows 2000-era, and 500 - Windows 7, so these numbers are super conservative.